client-core/src/synchronizer/auto_sync_data.rs

```diff
@@ -20,6 +20,17 @@ pub struct AutoSyncData {
     pub max_height: u64,
     /// current syncing wallet name
     pub wallet: String,
```

Comment:
Is it possible to also include a field of all the wallet being synced as well? I think this information is useful for the `sync_info` RPC

Reply:
ok